### PR TITLE
Display QGIS projects tags in LWC projects page for each project item

### PR DESCRIPTION
**Pull request by nboisteault**

### CODE REVIEW

File: lizmap/modules/view/templates/view.tpl [hunk 1]
```diff
@@ -19,6 +19,8 @@
           <br/>
           <br/><b>{@default.project.abstract.label@}</b>&nbsp;: {$p->abstract|strip_tags|truncate:100}
           <br/>
+          <br/><b>{@default.project.keywordList.label@}</b>&nbsp;: <span class="keywordList">{$p->keywordList}</span>
```
[the remark this answers is not in the document]
Comment (nboisteault) [reply lizmap/modules/view/templates/view.tpl]: It displays an empty string like other lines (abstract, ...)

**Comment by nboisteault** [comment]
@mdouchin You woud add this object in `view.js` ?

**Comment by nboisteault** [comment]
@mdouchin @rldhont Where do we put the keywords search input ? Next to actual search input ? Or we add a switcher next to actual input, to toggle between title and keywords mode ?

**Comment by mdouchin** [comment]
I would move the actual search input under the header, inside the main panel, centered horizontally and alway on top (say top:10px). Redundant search inputs would be awful IMHO

**Comment by nboisteault** [comment]
> I would move the actual search input under the header, inside the main panel, centered horizontally and alway on top (say top:10px). Redundant search inputs would be awful IMHO

So we have a switcher to toggle between modes ?

**Comment by mdouchin** [comment]
No, we need to discuss it. It is the occasion to re-think this feature.

**Pull request merged**